python_client/slicer_package_manager_client/cli.py

```diff
@@ -390,8 +408,11 @@ def _cli_deleteDraftRelease(sc, *args, **kwargs):
     """
     Delete a specific revision within the Draft release.
     """
-    rls = sc.deleteDraftRelease(*args, **kwargs)
-    print('%s %s (%s)\t%s' % (rls['name'], rls['meta']['revision'], rls['_id'], 'DELETED'))
+    try:
+        rls = sc.deleteDraftRelease(*args, **kwargs)
+        print('%s %s (%s)\t%s' % (rls['name'], rls['meta']['revision'], rls['_id'], 'DELETED'))
+    except Exception as exc_info:
```

Comment:
Catching `Exception` is usually bad practice. It prevent legitimate exception from being reported as error ...

Reply:
I suggest to also have a look at http://click.pocoo.org/5/exceptions/

Reply:
Well done. Much better